api/nodes/serializers.py

```diff
@@ -75,6 +76,20 @@ def update_institutions(node, new_institutions, user, post=False):
         node.add_affiliated_institution(inst, user)
 
 
+class RegionRelationshipField(RelationshipField):
+
+    def get_object(self, region_id):
+        try:
+            region = Region.objects.get(_id=region_id)
+        except Region.DoesNotExist:
+            raise exceptions.ValidationError(detail='Region {} is invalid.'.format(region_id))
+        return region
+
+    def to_internal_value(self, data):
+        region = self.get_object(data)
+        return {'region_id': region.id}
```

Comment:
This is still loading the entire region object. You can just load the id, i.e. using `.values`.